app/frontend/startup.js

```diff
@@ -29,8 +29,6 @@ const i18nController = require('./controllers/i18n_controller.js');
 const UiHelper = require('./helpers/ui_helper.js');
 window.uiHelper = new UiHelper();
 
-const { waitUntilIdle } = require('./helpers/ezra_helper.js');
-
```

Comment:
Not sure why these two lines were deleted, maybe by accident? Please restore.

Reply:
yes from the vs code ... when i had the conflict... I will revert it.